go-controller/pkg/node/bridgeconfig/bridgeflows.go

```diff
@@ -273,65 +273,120 @@ func (b *BridgeConfiguration) flowsForDefaultBridge(extraIPs []net.IP) ([]string
 	}
 	if ofPortPhys != "" {
 		for _, netConfig := range b.patchedNetConfigs() {
-			var actions string
+			var actions, actionsForExternallyRoutedTraffic string
 			if config.Gateway.Mode != config.GatewayModeLocal || config.Gateway.DisablePacketMTUCheck {
 				actions = fmt.Sprintf("output:%s", netConfig.OfPortPatch)
+				actionsForExternallyRoutedTraffic = "NORMAL"
 			} else {
 				// packets larger than known acceptable MTU need to go to kernel for
 				// potential fragmentation
 				// introduced specifically for replies to egress traffic not routed
 				// through the host
 				actions = fmt.Sprintf("check_pkt_larger(%d)->reg0[0],resubmit(,11)", maxPktLength)
+				actionsForExternallyRoutedTraffic = actions
 			}
 
 			if config.IPv4Mode {
-				// table 1, established and related connections in zone 64000 with ct_mark CtMarkOVN go to OVN
+				// table 1, tracked & established and tracked & related connections in zone 64000 with ct_mark matching
+				// this network's MasqCTMark and the bridge MAC as destination MAC go to OVN
 				dftFlows = append(dftFlows,
-					fmt.Sprintf("cookie=%s, priority=100, table=1, ip, ct_state=+trk+est, ct_mark=%s, "+
-						"actions=%s", nodetypes.DefaultOpenFlowCookie, netConfig.MasqCTMark, actions))
+					fmt.Sprintf("cookie=%s, priority=100, table=1, dl_dst=%s, ip, ct_state=+trk+est, ct_mark=%s, "+
```

Comment:
do we want this " " after the bridge's MAC here ? Is it harmful in any way ? 

Does somehow openflow become "cheaper" if we remove it ?